packages/calypso-products/src/feature-group-plan-map.ts

```diff
@@ -70,20 +69,22 @@ import { FeatureGroupMap } from './types';
 export const featureGroups: Partial< FeatureGroupMap > = {
 	[ FEATURE_GROUP_GENERAL_FEATURES ]: {
 		slug: FEATURE_GROUP_GENERAL_FEATURES,
```

Comment:
What do you think about renaming the slug to `FEATURE_GROUP_ESSENTIAL_FEATURES`? Might reduce confusion in the future. 🤔

Reply:
Yes good call I will do this.